perception/euclidean_cluster/src/voxel_grid_based_euclidean_cluster_node.cpp

```diff
@@ -26,22 +26,21 @@ VoxelGridBasedEuclideanClusterNode::VoxelGridBasedEuclideanClusterNode(
 {
   const bool use_height = this->declare_parameter("use_height", false);
   const int min_cluster_size = this->declare_parameter("min_cluster_size", 1);
-  const int max_cluster_size = this->declare_parameter("max_cluster_size", 500);
+  const int max_cluster_size = this->declare_parameter("max_cluster_size", 1024);
   const float tolerance = this->declare_parameter("tolerance", 1.0);
-  const float voxel_leaf_size = this->declare_parameter("voxel_leaf_size", 0.5);
+  const float voxel_leaf_size = this->declare_parameter("voxel_leaf_size", 0.01);
```

Comment:
Why so small?

Reply:
changed this back to original voxel leaf size, addressed in https://github.com/upowerrobotics/autoware.universe/pull/4/commits/674759e19779ac339680b668e3994527d1b09b4e